components/atom/input/src/index.js

```diff
@@ -8,11 +8,29 @@ import Password from './Password/index.js'
 import {INPUT_SHAPES, TYPES} from './config.js'
 
 const AtomInput = forwardRef(({type, ...props}, ref) => {
+  const checkIfValidNumberInput = event => {
```

Comment:
move it to settings to not redeclare the function on every declaration hoisting and simply get the outside defined

Reply:
Thank you! I moved the function to a helper.js file. Would it be preferred to have it in the config file or in a setting named file?

Reply:
=)